src/routes/files.ts

```diff
@@ -23,11 +23,11 @@ export default async function files(fastify: FastifyInstance) {
   fastify.get('/files/credentials', async (req, reply) => {
     const sessionCache: SessionCache = req.sessionCache;
     const results = new Map<ContactType, String[][]>();
+    const hierarchyProps = new Map<ContactType, string[]>();
```

Comment:
Not really, didn't see the need as i was only tracking two variables so it wouldn't make much of a difference. If we get to three variables though then it might be worth introducing a type for the result set